docs/src/modules/components/AppLayoutDocsFooter.js

```diff
@@ -109,6 +109,7 @@ async function postFeedbackOnSlack(data) {
     currentLocationURL: window.location.href,
     commmentSectionURL: `${window.location.origin}${window.location.pathname}#${commentedSection.hash}`,
     commmentSectionTitle: commentedSection.text,
+    githubRepo: process.env.SOURCE_CODE_REPO ?? 'https://github.com/mui/material-ui',
```

Comment:
This is a required variable of `withDocsInfra()`, I think we should assume it's here

```suggestion
    githubRepo: process.env.SOURCE_CODE_REPO,
```